packages/runtime/test-runtime-utils/src/generateToken.ts

```diff
@@ -25,8 +25,26 @@ import { v4 as uuid } from "uuid";
  */
 
 /**
- * Generates a JWT token to authorize access to a Routerlicious-based Fluid service. This function uses a browser
- * friendly auth library (jsrsasign) and should only be used in client (browser) context.
+ * Generates a {@link https://en.wikipedia.org/wiki/JSON_Web_Token | JSON Web Token} (JWT)
+ * to authorize access to a Routerlicious-based Fluid service.
+ *
+ * This function uses a browser friendly auth library ({@link https://www.npmjs.com/package/jsrsasign | jsrsasign})
+ * and should only be used in client (browser) context.
+ *
+ * @param tenantId - @see {@link @fluidframework/protocol-definitions#ITokenClaims.tenantId}
+ * @param key - TODO
```

Comment:
API key to authenticate user. (General comment: This usually comes paired with tenant, when you set up one)